cnf-tests/testsuites/pkg/networks/network.go

```diff
@@ -38,16 +38,17 @@ func (b *NetworkAttachmentDefinitionBuilder) WithTuning(sysctls string) *Network
 	return b
 }
 
-func (b *NetworkAttachmentDefinitionBuilder) WithBond(bondName, link1, link2 string) *NetworkAttachmentDefinitionBuilder {
+func (b *NetworkAttachmentDefinitionBuilder) WithBond(bondName, link1, link2 string, mtu int) *NetworkAttachmentDefinitionBuilder {
```

Comment:
This change is also in https://github.com/openshift-kni/cnf-features-deploy/pull/1129 ?

Reply:
Yes, will rebase whichever does not get in first.
Alternatively we could merge this one first, and rebase these two on top of it: #1174